include/circt/Dialect/FIRRTL/FIRRTLDeclarations.td

```diff
@@ -189,7 +189,7 @@ def MemOp : FIRRTLOp<"mem"> {
 
     /// Return the type of a port given the memory depth, type, and kind
     static BundleType getTypeForPort(uint64_t depth, FIRRTLType dataType,
-                                     PortKind portKind);
+                                     PortKind portKind, unsigned maskBits = 0);
```

Comment:
It looks like the `getMaskBits()` method you defined returns `size_t`, but this `maskBits` argument is an `unsigned`. Should the two match?

Reply:
Thanks for catching this, fixed.